packages/eslint-config-nlx/documentation.js

```diff
@@ -6,6 +6,13 @@ module.exports = {
     "jsdoc/require-jsdoc": [
       "error",
       {
+        require: {
+            FunctionDeclaration: true,
+            MethodDefinition: true,
+            ClassDeclaration: true,
```

Comment:
this changes the documentation linter to enforce documentation on any function definition (the default is just for `functino()`

fwiw `typedoc` the json / html / markdown generator already requires these, so it's mostly ensuring the two systems share a ruleset